app/app/fixtures/grants.json

```diff
@@ -213,7 +213,7 @@
       "real_period_seconds": "2592000",
       "frequency_unit": "days",
       "frequency": "30",
-      "token_address": "0x6A9865aDE2B6207dAAC49f8bCba9705dEB0B0e6D",
+      "token_address": "0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa",
       "token_symbol": "DAI",
       "gas_price": "25000.0000",
       "new_approve_tx_id": "0x980ce738853f454c83d5551dc086a29d2d89781fb041a9ce581474ffdf639de4",
```

Comment:
^ could we undo all these changes

Reply:
See above comment https://github.com/gitcoinco/web/pull/7212#discussion_r483652209 — I think we should keep these changes.

Reply:
gotcha